README.md

```diff
@@ -7,7 +7,7 @@ A mod loader for [Neos VR](https://neos.com/). Consider joining our community on
 If you are using the Steam version of Neos you are in the right place. If you are using the standalone version, read the [Neos Standalone Setup](doc/neos_standalone_setup.md) instructions. If you are on Linux, read the [Linux Notes](doc/linux.md).
 
 1. Download [NeosModLoader.dll](https://github.com/neos-modding-group/NeosModLoader/releases/latest/download/NeosModLoader.dll) to Neos's `Libraries` folder (`C:\Program Files (x86)\Steam\steamapps\common\NeosVR\Libraries`).
-2. Place [0Harmony.dll](https://github.com/neos-modding-group/NeosModLoader/releases/download/1.9.1/0Harmony.dll) into a `nml_libs` folder under your Neos install directory (`C:\Program Files (x86)\Steam\steamapps\common\NeosVR\nml_libs`). You will need to create this folder.
+2. Place [0Harmony.dll](https://github.com/neos-modding-group/NeosModLoader/releases/download/1.12.4/0Harmony.dll) into a `nml_libs` folder under your Neos install directory (`C:\Program Files (x86)\Steam\steamapps\common\NeosVR\nml_libs`). You will need to create this folder.
```

Comment:
My plan is to always have harmony attached to releases going forward, so as soon as 1.12.4 is marked as a "real" release and not a "pre" release that will take effect, and we should do this:

```suggestion
2. Place [0Harmony.dll](https://github.com/neos-modding-group/NeosModLoader/releases/latest/download/0Harmony.dll) into a `nml_libs` folder under your Neos install directory (`C:\Program Files (x86)\Steam\steamapps\common\NeosVR\nml_libs`). You will need to create this folder.
```